### PR TITLE
Update updating-dependencies.md

### PR DESCRIPTION
**Pull request by Shivkumar13**
Removing typos in updating-dependencies.md

### Which issue this PR addresses:

Fixes

### CODE REVIEW

File: docs/updating-dependencies.md [hunk 1]
```diff
@@ -30,10 +30,11 @@ semantic versioning via tags. Therefore the proper version is parsed from the ve
 branch and fixed using replace directive. Otherwise it will upgrade every time
 the command is started.
 
-When upgrading to a never version of OpenShift, this script have to be updated to
+When upgrading to a newer version of OpenShift, this script have to be updated to
 reflect the proper release.
 
 
+
```
Comment (facchettos) [inline docs/updating-dependencies.md:37]: is this needed for something or is this just something that accidentally got in?

**Review by facchettos — CHANGES_REQUESTED** [review]

File: docs/updating-dependencies.md (hunk 1 above)
Comment (Shivkumar13) [reply docs/updating-dependencies.md]: Accidentally got added, removed it!

**Review by facchettos — APPROVED** [review]

**Pull request merged**